Yoast/wordpress-seo
# Implement new presenters for Webmaster tools meta

## Context


* Implement new presenters for Webmaster tools meta

## Summary


This PR can be summarized in the following changelog entry:

* None needed; this replaces all the webmaster tools meta code with new presenters.

## Test instructions

This PR can be tested by following these steps:

* Set Google, Yandex, Bing, Baidu webmaster tools verification strings, verify that they are output correctly.
* Set Pinterest verification string under Social settings, verify that it is output correctly.

## Quality assurance


Fixes #14718
